lib/util/cache/repository/types.ts

```diff
@@ -32,6 +32,8 @@ export interface BranchCache {
 }
 
 export interface Cache {
+  configFileName?: string;
```

Comment:
I've decided not to cache the config *contents* for now, so I don't have to think about whether any secrets within need to be specially managed.

Reply:
Also no cache revision change necessary here because if these fields are not present then it's completely compatible